app/javascript/gobierto_dashboards/webapp/containers/summary/Summary.vue

```diff
@@ -116,11 +133,55 @@ export default {
       labelHalfSpendingsContracts: I18n.t('gobierto_dashboards.dashboards.contracts.summary.label_half_spendings_contracts'),
       labelContractType: I18n.t('gobierto_dashboards.dashboards.contracts.contract_type'),
       labelProcessType: I18n.t('gobierto_dashboards.dashboards.contracts.process_type'),
-      labelAmountDistribution: I18n.t('gobierto_dashboards.dashboards.contracts.amount_distribution')
+      labelAmountDistribution: I18n.t('gobierto_dashboards.dashboards.contracts.amount_distribution'),
+      labelMainAssignees: I18n.t('gobierto_dashboards.dashboards.contracts.main_assignees'),
+      labelTableThAssignee: I18n.t('gobierto_dashboards.dashboards.contracts.assignee'),
+      labelTableThContracts: I18n.t('gobierto_dashboards.dashboards.contracts.contracts'),
+      labelTableThAMount: I18n.t('gobierto_dashboards.dashboards.contracts.final_amount'),
     }
   },
+
   mounted() {
     EventBus.$emit("summary_ready");
+
+    EventBus.$on('refresh_summary_data', () => {
+      this.refreshSummaryData();
+    });
+  },
+  created() {
+    this.items = this.buildItems();
+    this.columns = assigneesColumns;
+  },
+  methods: {
+    refreshSummaryData(){
+      this.contractsData = this.$root.$data.contractsData;
+      this.items = this.buildItems();
+    },
+    buildItems() {
+      const groupedByAssignee = {}
+      // Group contracts by assignee
+      this.contractsData.forEach((contract) => {
+        if (contract.assignee === '' || contract.assignee === undefined) {
+          return;
+        }
+
+        groupedByAssignee[contract.assignee] = groupedByAssignee[contract.assignee] || {name: contract.assignee}
+
+        // vue wouldn't update the table rows if there is no id attribute
+        groupedByAssignee[contract.assignee].id = groupedByAssignee[contract.assignee].id || uuid()
+
+        groupedByAssignee[contract.assignee].sum = groupedByAssignee[contract.assignee].sum || 0
+        groupedByAssignee[contract.assignee].sum += parseFloat(contract.final_amount)
+
+        groupedByAssignee[contract.assignee].count = groupedByAssignee[contract.assignee].count || 0
+        groupedByAssignee[contract.assignee].count++;
+      });
```

Comment:
Do you see how contract is yelling: somebody destructure meeeee?? (and initialize meeee)

Reply:
😂  

man, it was pretty obvious. I don't know how could I miss that.